### PR TITLE
Add more visually striking marketplace menu item in sidebar

### PR DESCRIPTION
**Pull request by caddoo**
Original PR:
https://github.com/matomo-org/matomo/pull/21456

Tested locally, updated and looked through all UI tests that failed and recommitted because of the differences between 4 and 5.

---------

### Description:

Please include a description of this change and which issue it fixes. If no issue exists yet please include context and what problem it solves.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png
Comment (bx80) [inline plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png:1]: Were we expecting a change here? The layout seems to have moved up by a couple of pixels.

File: tests/UI/expected-screenshots/IncompletePeriodVisualisation_visitors_overview.png
Comment (bx80) [inline tests/UI/expected-screenshots/IncompletePeriodVisualisation_visitors_overview.png:1]: Similar, is change expected?

File: tests/UI/expected-screenshots/QuickAccess_search_1.png
Comment (bx80) [inline tests/UI/expected-screenshots/QuickAccess_search_1.png:1]: The marketplace entry now seems to be missing from the quick access search.

**Review by bx80 — CHANGES_REQUESTED** [review]
The changes look good, but there are few unexpected(?) test screenshot changes that might need checking.

File: tests/UI/expected-screenshots/QuickAccess_search_1.png
Comment (caddoo) [reply tests/UI/expected-screenshots/QuickAccess_search_1.png:1]: That is accepted/expected for now was discussed between @michalkleiner and @Stan-vw already.

File: tests/UI/expected-screenshots/IncompletePeriodVisualisation_visitors_overview.png
Comment (caddoo) [reply tests/UI/expected-screenshots/IncompletePeriodVisualisation_visitors_overview.png:1]: Nope, I've reverted now, and can confirm this is a flaky test within testomat

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png
Comment (caddoo) [reply plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png:1]: Nope, I've reverted now, and can confirm this is a flaky test within testomat

**Review by bx80 — APPROVED** [review]

**Pull request merged**